src/audio/volume/volume_hifi4.c

```diff
@@ -21,7 +21,7 @@ LOG_MODULE_DECLARE(volume_hifi4, CONFIG_SOF_LOG_LEVEL);
 
 #include "volume.h"
 
-#ifdef VOLUME_HIFI4
+#if SOF_USE_HIFI(4, VOLUME)
```

Comment:
Ask @andrula-song to confirm but I think you need this until volume_hifi5.c exists.
```suggestion
#if SOF_USE_HIFI(4, VOLUME) || SOF_USE_HIFI(5, VOLUME)
```